consumer/schemas/inventory.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class Device(BaseModel):
+    id: int
+    serial: str
+    device_id: str
+    is_checkout: bool
+    model: Optional[str]
+    manufacturer: Optional[str]
+    location: Optional[str]
+
+    @classmethod
+    def serialize(cls, device: dict) -> "Device":
+        """Simplifies reuse across inventory API."""
+
+        def name_or_none(item: dict) -> Optional[str]:
+            return item.get("name", None) if item else None
+
+        return cls(
+            id=device["id"],
+            serial=device["serial"].replace(" ", ""),
+            device_id=device["asset_tag"],
+            is_checkout=device["status_label"]["status_meta"] == "deployed",
+            model=name_or_none(device["model"]),
+            manufacturer=name_or_none(device["manufacturer"]),
+            location=name_or_none(device["location"]),
+        )
+
+
+class HistoryItem(BaseModel):
+    patient_id: str
+    datetime: str
+
+    @classmethod
+    def serialize(cls, device: dict) -> "HistoryItem":
+        """Device is an item from a snipe-it response.
+        This also contains an 'action_type' that is either 'checkout'
+        or 'checkin from' but is not currently used."""
+        return cls(
+            patient_id=device["target"]["name"].strip(),
```

Comment:
You could use the `dict.get("name")` here too if applicable.

Reply:
I agree, but in this case we probs don't want a default name (patient_id) so letting it throw if it doesn't exist may be ideal?